napari/_tests/test_with_screenshot.py

```diff
@@ -313,9 +314,9 @@ def test_grid_mode(make_test_viewer):
     # return to stack view
     viewer.grid.enabled = False
     assert not viewer.grid.enabled
-    assert viewer.grid.actual_size(6) == (1, 1)
+    assert viewer.grid.actual_shape(6) == (1, 1)
     assert viewer.grid.stride == 1
-    translations = [layer.translate_grid for layer in viewer.layers]
+    translations = [vis.translate_grid[:2][::-1] for vis in visuals]
```

Comment:
What is `actual_shape` compared to `shape`? This API might benefit from a rethink...

Reply:
and `vis.translate_grid[:2][::-1]` is super mysterious here...